package.json

```diff
@@ -20,6 +20,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-redux": "^8.0.5",
+    "sass": "^1.58.0",
```

Comment:
why have we added this?

Reply:
/navbar.module.scss (style of navbar in sass format)

Reply:
we don't need to do that